components/MapComponent.vue

```diff
@@ -237,9 +237,9 @@
        this.timelineRangeStart = startRange;
```

Comment:
The main map doesn't load for me just now. Is that to be expected or should I go hunting for a config issue? Seeing "LMap" is missing in my console so thinking this is deliberately broken at the moment?

Reply:
This works in the cluster map so I think this is ok. Marking as resolved